### ISSUE TITLE
Question about data processing

### ISSUE THREAD

**Issue by kkk00714** [Amir-Samadi/VVF-TP#1]
Hello, I would like to know some specific steps about data set processing, such as what the data set goes through before it is divided into a training set/test set/verification set, and how many size sliding Windows you use to sample the same car. Thank you very much! @Amir-Samadi

**Comment by tjx97ngddr** [comment]
> 您好，我想了解一些关于数据集处理的具体步骤，例如数据集在被划分为训练集/测试集/验证集之前要经过什么，以及您使用多少个尺寸的滑动窗口来对同一辆车进行采样。谢谢！
@kkk00714 ，您好，您最后这个代码跑通了吗？我想了了解一下，数据集再测试之前得操作，也就是如何将数据集转换为.h5得格式。

**Comment by kkk00714** [comment]
> > 您好，我想了解一些关于数据集处理的具体步骤，例如数据集在被划分为训练集/测试集/验证集之前要经过什么，以及您使用多少个尺寸的滑动窗口来对同一辆车进行采样。谢谢！
> > @kkk00714 ，您好，您最后这个代码跑通了吗？我想了了解一下，数据集再测试之前得操作，也就是如何将数据集转换为.h5得格式。

我并未跑这个代码，因为我只是想了解相关工作对highD的数据预处理步骤。我的滑动窗口是15帧，训练集/测试集/验证集的比例为0.7/0.1/0.2，并且这个比例是针对于每个独立的车辆的，假如一共有10000辆车，那么随机抽选的7000辆车的所有轨迹将会成为训练集。
